Komga: Added README file (#10566)
tachiyomiorg/tachiyomi-extensions: src/all/komga/README.md

# Komga

Table of Content
- [FAQ](#FAQ)
- [Guides](#Guides)

Don't find the question you are look for go check out our general FAQs and Guides over at [Extension FAQ](https://tachiyomi.org/help/faq/#extensions) or [Getting Started](https://tachiyomi.org/help/guides/getting-started/#installation)

## FAQ

> ### Why do i see no manga?

Reviewer: ```suggestion
### Why do I see no manga?
```

Komga is a self-hosted comic/manga media server.

> ### Where can i get more information about Komga?

Reviewer: ```suggestion
### Where can I get more information about Komga?
```

You can visit the [Komga](https://komga.org/) website for for more information.

### The Komga extension stopped working?
Make sure that your Komga server and extension are on the newest version.

> ### Can i add more than one Komga server or user?

Reviewer: ```suggestion
### Can I add more than one Komga server or user?
```

Yes, currently you can add up to 3 different Komga instances to Tachiyomi.

## Guides

> ### How do i add my Komga server to Tachiyomi?

Reviewer: ```suggestion
### How do I add my Komga server to Tachiyomi?
```

> Go into the setting menu of the Komga extension and fill in your server address and login details.

Reviewer: ```suggestion
Go into the settings of the Komga extension from the Extension tab in Browse and fill in your server address and login details.
```

Author: Can't change it in this PR as its already merged would require a new PR.

Reviewer 2: Make a new one, and combine a few readmes.  I'll let it simmer for a day so people can review then merge.
